server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -66,6 +67,12 @@ func (b *BuildEventHandler) OpenChannel(ctx context.Context, iid string) interfa
 		targetTracker:           target_tracker.NewTargetTracker(b.env, buildEventAccumulator),
 		hasReceivedStartedEvent: false,
 		eventsBeforeStarted:     make([]*inpb.InvocationEvent, 0),
+		logWriter: chunkstore.New(
+			b.env.GetBlobstore(),
+			&chunkstore.ChunkstoreOptions{
+				WriteBlockSize:       (1 << 20) * 2,
+				WriteTimeoutDuration: 15 * time.Second,
+			}).Writer(ctx, iid+"/chunks/log/eventlog"),
```

Comment:
nit: we should probably define a factory function for assembling these remote blobstore "paths" so you don't end up with 2 different paths because of a typo

Reply:
good point.